editor/src/core/model/scene-id-utils.ts

```diff
@@ -11,10 +11,10 @@ import * as PP from '../shared/property-path'
 import { assertNever } from '../shared/utils'
 import { isSceneAgainstImports, isRemixSceneAgainstImports } from './project-file-utils'
 
-const IdPropName = 'id'
+export const SceneCommentIdPropName = 'commentId'
```

Comment:
small suggestion - maybe `comment-target-id`?
(since it's not the id of a comment, but the id of the scene as a comment target)